src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -19,30 +20,44 @@ type AttachmentCarouselPagerHandle = {
     setPage: (selectedPage: number) => void;
 };
 
-type PagerItem = {
-    key: string;
-    url: string;
+type Attachment = {
     source: string;
 };
 
 type AttachmentCarouselPagerProps = {
-    items: PagerItem[];
-    renderItem: (props: {item: PagerItem; index: number; isActive: boolean}) => React.ReactNode;
-    initialIndex: number;
+    /**
+     * The attachments to be rendered in the pager.
+     */
+    items: Attachment[];
+    /**
+     * The source (URL) of the currently active attachment.
+     */
+    activeSource: string;
+    /**
+     * The index of the initial page to be rendered.
+     */
+    initialPage: number;
+    /**
+     * A callback to be called when the page is changed.
+     */
     onPageSelected: () => void;
+    /**
+     * A callback that can be used to toggle the attachment carousel arrows, when the scale of the image changes.
+     * @param showArrows If set, it will show/hide the arrows. If not set, it will toggle the arrows.
+     */
```

Comment:
```suggestion

    /**
     * The source (URL) of the currently active attachment.
     */
    activeSource: string;

    /**
     * The index of the initial page to be rendered.
     */
    initialPage: number;

    /**
     * A callback to be called when the page is changed.
     */
    onPageSelected: () => void;

    /**
     * A callback that can be used to toggle the attachment carousel arrows, when the scale of the image changes.
     * @param showArrows If set, it will show/hide the arrows. If not set, it will toggle the arrows.
     */
```